templates/common/baremetal/files/baremetal-keepalived.yaml

```diff
@@ -83,8 +83,8 @@ contents:
             - /bin/sh
```

Comment:
Oh, this is naughty when we're using [[]], which is a bash-ism that's not present in a plain POSIX shell. This should be /bin/bash.